### DISCUSSION TITLE
Org-block face overlaps vertico-current face

### DISCUSSION THREAD

**Discussion by Doodler8888** [minad/consult#1075] [Q&A] [answered]
Is there any way to make the vertico-current face to overlap the org-block face?

![image](https://github.com/user-attachments/assets/c71c392c-291d-4b54-8df9-d85813ce2e17)

like how it works in swiper:

![image](https://github.com/user-attachments/assets/38579147-c180-4e1e-a905-ca0bad0075eb)

**Comment by minad** [comment, marked as answer]
The face marking the current candidate is appended, see https://github.com/minad/vertico/blob/d70fdeb67be9ecc88c438039eefa9ef0e2104005/vertico.el#L507. I don't recall the exact reasons for appending, but I guess it led to fewer problems. You could try prepending and check if it works better for you. The alternative would be a more complex face merging mechanism, which is used afaik by Ivy/Swiper, where color blending is used. Such a mechanism is out of scope of my packages, since it would interfere with the colors chosen by the theme.
